bin/check_version

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+#
+# This script intentionally runs silent if there is no version mismatch found.
+#
+
+ROOT_DIR=$(dirname $0)/..
+cd $ROOT_DIR
+
+branch=master
+origin=`git remote -v | fgrep faucetsdn/udmi | awk '{print $1}' | uniq`
```

Comment:
TBH I'm a little surprised that we can't do tihs a little more explicitly with git remote, but, at a quick glance it seems not. No problem for now though.